docs/developers/packaging.md

```diff
@@ -120,6 +142,8 @@ pip install dist/some_name.whl
 
 Upload package:
 
+# TODO
```

Comment:
Is this on purpose?

Reply:
No, it isn't. I forgot to remove it. Thanks.